.renovaterc.json

```diff
@@ -23,6 +23,60 @@
       "matchPackageNames": ["typescript"],
       "allowedVersions": "4.7.4",
       "matchPaths": ["+(package.json)"]
+    },
+    {
+      "groupName": "monorepo:supernovaio non-major",
+      "groupSlug": "monorepo-supernovaio-non-major",
+      "matchPackagePatterns": ["^@supernovaio"],
+      "matchUpdateTypes": ["minor", "patch"]
+    },
+    {
+      "groupName": "eslint plugins and configs non-major",
+      "groupSlug": "eslint-plugins-configs-non-major",
+      "matchPackagePatterns": ["^eslint"],
+      "matchUpdateTypes": ["minor", "patch"]
+    },
+    {
+      "groupName": "compile tools non-major",
+      "groupSlug": "compile-tools-non-major",
+      "matchPackagePatterns": ["^@babel", "^@swc"],
+      "matchUpdateTypes": ["minor", "patch"]
+    },
+    {
+      "groupName": "types non-major",
+      "groupSlug": "types-non-major",
+      "matchPackagePatterns": ["^@types"],
+      "matchUpdateTypes": ["minor", "patch"]
+    },
+    {
+      "groupName": "monorepo:react non-major",
+      "groupSlug": "monorepo-react-non-major",
+      "matchPackagePatterns": ["^react"],
+      "matchUpdateTypes": ["minor", "patch"]
+    },
+    {
+      "groupName": "lint tools non-major",
+      "groupSlug": "lint-tools-non-major",
+      "matchPackagePatterns": ["^stylelint", "^prettier", "^lint-staged", "^husky"],
```

Comment:
Since you have ESlint in a separate group, we can also have a group for Stylelint. @crishpeen or me could then take care of it.

Reply:
Well, it could be. But from my experience, there is a bigger chance of having an issue with the ESLint and all of the plugins than with these four packages. That is why I group it this way.

Reply:
I see what you mean. I'm just saying that if you move Stylelint to its own group, I am open to look after the updates because it's closer to my "sandbox". For example, should there be an update of Stylelint _and_ `lint-staged` or `husky`, I will probably not in rush to fix that because you understand the last two better 🙂.

Reply:
Ok, I have separated it.